packages/wrangler/src/init.ts

```diff
@@ -36,6 +40,12 @@ export async function initOptions(yargs: Argv) {
 			describe: 'Answer "yes" to any prompts for new projects',
 			type: "boolean",
 			alias: "y",
+		})
+		.option("from-dash", {
+			describe: "Download script from the dashboard for local development",
+			type: "string",
+			requiresArg: true,
+			hidden: true,
```

Comment:
?

Reply:
To clarify: why is this option hidden?

Reply:
While its in development and "experimental" (not finished) it will be hidden.

See: https://github.com/cloudflare/wrangler2/discussions/1623#discussioncomment-3334992